lib/OptimalBranchingMIS/src/tablesolver.jl

```diff
@@ -90,6 +101,16 @@ function prune_by_env(tbl::BranchingTable{INT}, p::MISProblem, vertices) where{I
             push!(new_table, tbl.table[i])
         end
     end
-
     return BranchingTable(OptimalBranchingCore.nbits(tbl), new_table)
+end
+
+# Calculate the MWIS size for small-scale graphs using tensor network contraction, whose contraction order is greedily searched.
+function small_scale_mis_size(g::SimpleGraph{Int}, weights::Vector{WT}) where WT
+    problem = GenericTensorNetwork(IndependentSet(g, weights); optimizer = GreedyMethod(nrepeat=1))
+    return solve(problem, SizeMax())[].n
+end
+
+# Calculate the MIS size for small-scale graphs using mis2 algorithm.
+function small_scale_mis_size(g::SimpleGraph{Int}, weights::UnitWeight)
```

Comment:
Can we not using `mis2`? BTW, this function is available here: https://github.com/QuEraComputing/GenericTensorNetworks.jl/blob/0942d7a71063a76580b5b636b776e7ef9d2a5c6d/docs/src/gist.md?plain=1#L51